testing/bullet/APKBUILD

```diff
@@ -0,0 +1,63 @@
+# Contributor: Danct12 <danct12@disroot.org>
+# Contributor: Asriel Dreemurr <asriel.danctnix@gmail.com>
+# Maintainer: Danct12 <danct12@disroot.org>
+pkgname=bullet
+pkgver=2.88
+pkgrel=0
+pkgdesc="A 3D Collision Detection and Rigid Body Dynamics Library for games and animation"
+url="https://pybullet.org/Bullet/phpBB3/"
+arch="all"
+license="Zlib"
+makedepends="cmake doxygen graphviz-dev ttf-dejavu mesa-dev glu-dev python3-dev py-numpy-dev"
+options="!check" # cannot build tests
+subpackages="$pkgname-dev $pkgname-doc"
+source="bullet3-$pkgver.tar.gz::https://github.com/bulletphysics/bullet3/archive/$pkgver.tar.gz"
+builddir="$srcdir/bullet3-$pkgver"
+
+prepare() {
+	default_prepare
+	mkdir "$builddir"/build
+}
+
+build() {
+	cd "$builddir"/build
+
+	cmake .. \
+		-DCMAKE_INSTALL_PREFIX=/usr \
+		-DBUILD_SHARED_LIBS=1 \
+		-DBUILD_BULLET2_DEMOS=0 \
+		-DBUILD_CPU_DEMOS=0 \
+		-DINSTALL_LIBS=1 \
+		-DINSTALL_EXTRA_LIBS=1 \
+		-DBUILD_PYBULLET=ON \
+		-DBUILD_PYBULLET_NUMPY=ON \
+		-DBUILD_OPENGL3_DEMOS=0 \
+		-DBUILD_UNIT_TESTS=0 \
+		-DCMAKE_BUILD_TYPE=Release
+
+	make
+
+	cd ..
+	doxygen
+}
+
+package() {
+	cd "$builddir"/build
+	make -j1 DESTDIR="$pkgdir" install	# avoid libtool errors
+
+	# Install LICENSE
+	install -Dm644 ../LICENSE.txt $pkgdir/usr/share/licenses/$pkgname/LICENSE
+}
+
+doc() {
+	pkgdesc="Documentation for bullet"
```

Comment:
You should call `default_doc` if you override the doc function. Then you don't need to manually set the `pkgdesc` either.

Reply:
Done that.